__tests__/shared/credentialStatus.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  CredentialPayload, IAgentOptions,
+  IDataStore,
+  IDataStoreORM,
+  IDIDManager,
+  IIdentifier,
+  TAgent
+} from '@veramo/core/src'
+import { CredentialStatusPlugin } from '../../packages/credential-status/src/credential-status'
+import { ICredentialIssuer } from '../../packages/credential-w3c/src'
+
+type ConfiguredAgent = TAgent<IDIDManager & ICredentialIssuer & IDataStore & IDataStoreORM>
+
+export default (testContext: {
+  getAgent: () => ConfiguredAgent
+  setup: (options?: IAgentOptions) => Promise<boolean>
+  tearDown: () => Promise<boolean>
+}) => {
+  describe('checking Verifiable Credentials status (revocation)', () => {
+    let agent: ConfiguredAgent
+    let identifier: IIdentifier
+    let statusChecked: boolean
+    let rawCredential: CredentialPayload
+
+    beforeAll(async () => {
+      const expectedResult = {}
+      statusChecked = false
+      const checkStatus = jest.fn(async (credential) => {
```

Comment:
Actually, the `checkStatus` is called inside the `CredentialStatusPlugin` and not in the test itself, that's why I used this pattern instead of a simples verification.